tests/README.md

```diff
@@ -0,0 +1,32 @@
+# Test for git-extras
+The git-extras has its own testcases now, and the more is on the way! So let's introduce it.
+
+We choose python to help us to reach to other shore cause **python is life saver**.
+
+The test part depends on:
+
+* python >= 3.11
```

Comment:
Should duplicate information be removed from the `Testing` section under the `CONTRIBUTING.md`?

Reply:
Ya..I misremember the test content written before. I just wonder why i cannot find it 🤣